content/applications/sales/point_of_sale/shop/posreceipts.rst

```diff
@@ -0,0 +1,37 @@
+========
+Receipts
+========
+
+Configuration
+=============
+
+Set up receipts by going to :menuselection:`Point of Sale --> Configuration --> Point of Sale`,
+selecting a POS, and scrolling down to the :guilabel:`Bills & Receipts` section.
+
+To **customize** the **header** and **footer**, activate :guilabel:`Header & Footer`. Doing so
+enables fields to fill in with the information you want to be printed on the receipts. To **print
+receipts** automatically once the payment is registered, enable the :guilabel:`Automatic Receipt
+Printing` setting.
+
+.. image:: receipts/receipt.png
+   :align: center
+   :alt: pos receipt
+
+.. seealso::
+   - :doc:`../restaurant/bill_printing`
+   - :doc:`../restaurant/split`
+
+Reprint a receipt
+=================
+
+From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the
```

Comment:
```suggestion
From the POS interface, click :guilabel:`Orders` at the top, open the dropdown selection menu next to the
```

Reply:
it would suggest there's another "orders" on the POS interface, I don't think it's necessary to precise where the order button is situated